nuxeo-core/nuxeo-core-mimetype/src/test/java/org/nuxeo/ecm/platform/mimetype/TestMimetypeRegistryService.java

```diff
@@ -60,33 +58,25 @@
 @Deploy("org.nuxeo.ecm.core.mimetype")
 public class TestMimetypeRegistryService {
 
-    public static final String APPLICATION_OCTET_STREAM = "application/octet-stream";
+    /**
+     * @deprecated since 11.1. Use {@link MimetypeRegistry#DEFAULT_MIMETYPE} instead.
+     */
+    @Deprecated(since = "11.1", forRemoval = true)
+    public static final String APPLICATION_OCTET_STREAM = MimetypeRegistry.DEFAULT_MIMETYPE;
```

Comment:
If you deprecate, please update all uses of the deprecated constant.
Yet, I think that we can safely remove the constant without deprecating it, and add a static import of `MimetypeRegistry.DEFAULT_MIMETYPE;`.
To be checked with the other team members.